src/pages/docs/individuals/index.md

```diff
@@ -276,6 +278,14 @@ This is an example of a response to this url: `http://localhost:8084/my-project/
 
 ```
 
+Aggregations of individuals by a field are by default sorted by the number of individuals grouped 
+(aggregations with a higher number of individuals first), but they can also be
+ordered by the value of the field (See [sorting](/docs/general#sorting)). 
```

Comment:
I'd just add  in "ordered by the value of the field" this "ordered by the value of the grouping field"
Note: Only this field could be used, otherwise the default sort (doc_count desc) will be set